packages/textfield/src/Textfield.ts

```diff
@@ -225,7 +225,7 @@ export class TextfieldBase extends ManageHelpText(
 
     private get renderMultiline(): TemplateResult {
         return html`
-            ${this.grows && !this.quiet && this.rows === -1
+            ${this.grows && this.rows === -1
```

Comment:
Why are you checking "this.rows === -1"?

Reply:
When added, we decided that `rows` was more important in the attribute application workflow than `grows`, the implication being, if you set a specific number of `rows`, it should stay that many.

This reminded me to check if that was true/worked, and it appears not to... I'll do some tweaking of the CSS and add a new story to confirm that going forward.

Reply:
Okay!
So basically by "rows===-1" the code is implying that this textarea does not have fixed number of rows.
Right?

Reply:
Yep!